README.md

```diff
@@ -32,11 +32,12 @@ require 'vendor/autoload.php';
 
 First you need to get a fresh OAuth2 access token:
 
-...
+> See the [Authentication](#authentication) part below to see how.
 
 Instance a new client with the token:
 
 ```php
+<?php 
```

Comment:
but required for syntaxic coloration on some git Apps (gitlab AFAI) or markdown readers